numpy/polynomial/laguerre.py

```diff
@@ -1595,6 +1657,13 @@ def lagweight(x):
 
     .. versionadded:: 1.7.0
 
+    Examples
+    --------
+    >>> from numpy.polynomial.laguerre import lagweight
+    >>> x = np.array([0, 1, 2])
+    >>> lagweight()
```

Comment:
```suggestion
    >>> lagweight(x)
```
Missing argument to `lagweight`. The CI tests pass, so this means there are no doctests running. That might be something to consider.

Reply:
You are right! This might be something to look at for the maintainers?

Reply:
I managed to reproduce it locally. 
It looks like `PUBLIC_SUBMODULES` in `refguide_check.py` is missing a bunch of submodules [link](https://github.com/numpy/numpy/blob/35c4319f33f3eafa091d4c9790b6de2468874b51/tools/refguide_check.py#L62C6-L62C6) (it doesn't discover them recursively, they need to be listed explicitly, just as [here](https://github.com/numpy/numpy/blob/35c4319f33f3eafa091d4c9790b6de2468874b51/numpy/tests/test_public_api.py#L110)).

Reply:
Opened a PR here: https://github.com/numpy/numpy/pull/25367